content/rs/references/procedures-previous-releases.md

```diff
@@ -9,6 +9,41 @@ Here you can find procedures that were used in previous releases.
 
 ## Updating SSL/TLS certificates for Previous Releases
 
+### Before 5.4.4
```

Comment:
@IAmATeaPot418 It looks from the words "in current versions" that this command is good for after 5.4.4 also. Can you clairfy?

Reply:
@bmansheim - yes this is good in 5.4.4 where it was initially tested.

Reply:
@Paz-Yanover - any issues with this PR?